.gitignore

```diff
@@ -10,3 +10,4 @@
 *.iml
 /designs/
 interchange/schema/capnp/java.capnp
+*.orig
```

Comment:
Is this necessary?

Reply:
Oops, this is an unrelated change: IntelliJ likes to create these temporary files when handling merge conflicts. I can remove it if you oppose this change.